### PR TITLE
Feature/remove dependency on flutter

### PR DESCRIPTION
**Pull request by defuncart**
Thank you for creating this package. For dart packages, the dependency on flutter isn't desirable and unnecessary. Thus, this MR removes the dependency on flutter.

unit tests can be ran using `pub run test test/web_scraper_test.dart `, while example compiles and functions on iOS and Android devices.

### CODE REVIEW

File: test/web_scraper_test.dart [hunk 1]
```diff
@@ -19,10 +18,9 @@ void main() {
       expect(timeElapsed, isNotNull);
     });
     test('Parse tags', () async {
-      productNames = webScraper.getElement(
-          'div.thumbnail > div.caption > h4 > a.title', ['href', 'title']);
+      productNames = webScraper.getElement('div.thumbnail > div.caption > h4 > a.title', ['href', 'title']);
       expect(productNames, isNotNull);
-    });
```
Comment (tusharojha) [inline test/web_scraper_test.dart:25]: Please format all dart files using the `flutter format` command.

**Comment by tusharojha** [comment]
Hi @defuncart, 
Everything is good and I have a few things if you can do:

- Format all dart files (since the Travis CI is not yet added, please check manually)
- Since its a major change, It is necessary to mention in the `Readme` file. 

Also, the master branch is synced with `pub.dev`, therefore a new feature should be added to the `develop` branch first. This time we can merge it directly but it will be great if you mention this also in the Readme.

Thanks for your contribution~

File: test/web_scraper_test.dart (hunk 1 above)
Comment (defuncart) [reply test/web_scraper_test.dart]: Done :)

**Comment by defuncart** [comment]
Hi @tusharojha. I can change the PR target to be develop if you like. If the target should remain as master, should I update the version to 0.0.6? This isn't actually a major change, and should not affect any current users IMO. Functionality is the exact same as before.

With regard to CHANGELOG, I can note the removal of Flutter as a dependency.

**Comment by tusharojha** [comment]
@defuncart I think we can push to version 0.0.6. 

I would like to request the following changes:
- Update the version to 0.0.6
- Update `Readme.md` file with a major change notice and also add information about `develop` branch in the contribution section
- Update the `changelog.md` with the new changes

**Comment by defuncart** [comment]
@tusharojha Please see latest commit.

**Pull request merged**